### PR TITLE
Remove publishing tasks from build

### PR DESCRIPTION
**Pull request by murthys**
Remove publishing tasks from build pipeline, as those will run in release pipeline instead.
Also delete package-lock.json which was added by mistake in last PR.

### CODE REVIEW

**Comment by erinha** [comment]
$(bundleAnalysisTask.bundleAnalysisComment)

**Comment by erinha** [comment]
$(bundleAnalysisTask.bundleAnalysisComment)

**Comment by erinha** [comment]
$(bundleAnalysisTask.bundleAnalysisComment)

**Comment by erinha** [comment]
$(bundleAnalysisTask.bundleAnalysisComment)

File: azure-pipelines.yml [hunk 1]
```diff
@@ -256,14 +248,6 @@ steps:
       flattenFolders: true
     displayName: 'Copy JS Content for NPM'
 
-  - task: CopyFiles@2
```
Comment (murthys) [inline azure-pipelines.yml:259]: We dont need this script in release pipeline as, as we are using the Out of the Box task in release pipeline to publish

**Review by ramourt — APPROVED** [review]
lgtm

**Pull request merged**